index.html

```diff
@@ -5402,32 +5402,32 @@ <h3>Promote and implement responsible emerging technology practices</h3>
 				<section class="notoc" data-testable="machine" data-impact="high" data-effort="medium" data-materials="high" data-energy="high" data-water="high" data-emissions="high" data-standard="SDGs" data-considerations="Privacy, Security" data-categories="Content, E-Waste, Governance, Hardware, Networking, Performance, Social Equity, Software">
 					<h4 id="emerging-technologies">Success Criterion: Emerging technologies</h4>
 					<p class="external"><a class="machine" href="https://w3c.github.io/sustainableweb-wsg/star.html#BSPM22-1">Machine-testable</a> <span class="hide">and </span><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#BSPM22-1">Resources</a></p>
-					<p>The organization has public-facing policies in place for emerging technologies, and all such technologies are ethically sourced, screened, validated, and implemented in a non-discriminatory, responsible manner.</p>
+					<p>Establish public-facing policies for emerging technologies. Ensure all such technologies and their datasets are ethically sourced, screened, validated, and implemented in a non-discriminatory, responsible manner.</p>
 				</section>
 				<section class="notoc" data-testable="human" data-impact="high" data-effort="medium" data-materials="high" data-energy="high" data-water="high" data-emissions="high" data-standard="SDGs" data-considerations="Privacy, Security" data-categories="Content, E-Waste, Governance, Hardware, Networking, Performance, Social Equity, Software">
 					<h4 id="disruptive-technology">Success Criterion: Disruptive technology</h4>
 					<p class="external"><span class="human">Human-testable</span> <span class="hide">and </span> <a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#BSPM22-2">Resources</a></p>
-					<p>The organization shows how it up-skills workers as new technologies and practices potentially disrupt its business model.</p>
+					<p>Show how workers are trained as new technologies and practices potentially disrupt its business model.</p>
```

Comment:
The "it's" in this sentence is out of place now after the changes made. Maybe "... potentially disrupt organisational business models." or something similar.